sdk/eventhub/eventhubs-checkpointstore-blob/samples/downlevelContainerClient.js

```diff
@@ -0,0 +1,42 @@
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+  This sample demonstrates how to extend the Storage Blob Container Client
```

Comment:
Change the language to something like "This sample demonstrates how to extend the Storage Blob Container Client to change the level of its API version used when communicating with the service."